source/adapters/level_zero/queue.cpp

```diff
@@ -1192,10 +1192,20 @@ ur_queue_handle_t_::ur_queue_handle_t_(
     }
     return std::atoi(UrRet) != 0;
   }();
+  static const bool useInterruptBasedEvents = [] {
+    const char *UrRet = std::getenv("UR_L0_USE_INTERRUPT_BASED_EVENTS");
```

Comment:
I don't think the env is necessary, this event type creation is based on input flags from the user. They are not automatically created and should only event be created when the user requested it.

Reply:
Gotcha, removed.